### PR TITLE
chore: SIMS code added to solution

### PR DESCRIPTION
**Pull request by milana-todorovic**
@PSW-2020-ORG4/devops Check if all the added projects build and run correctly. Most of the existing code needs to be refactored, but that was ignored for now.

### CODE REVIEW

**Comment by milana-todorovic** [comment]
[(2) Add old code to repository [d2] - 1h 47min](https://trello.com/c/Sn13qhLo/46-2-add-old-code-to-repository-d2-1h-47min)

**Review by vlaksi — APPROVED** [review]
Lgtm, but you need to build it twice. 😸

**Review by anciz — APPROVED** [review]
Everything is OK.

File: HealthcareSystem/Backend/Controller/NotificationSurveyAndFeedback/FeedbackController.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      public Model.Users.Feedback AddFeedback(Feedback feedback)
+      {
+            return feedbackService.AddFeedback(feedback);
+      }
+      
+      public List<Model.Users.Feedback> ViewFeedbackInformations()
+      {
+            return feedbackService.ViewFeedbackInformations();
+      }
+
```
Comment (vlaksi) [inline HealthcareSystem/Backend/Controller/NotificationSurveyAndFeedback/FeedbackController.cs:27]: lgtm

**Pull request merged**